lib/active_model/serializer.rb

```diff
@@ -84,11 +85,14 @@ def self.serializers_cache
     #   1. class name appended with "Serializer"
     #   2. try again with superclass, if present
     #   3. nil
-    def self.get_serializer_for(klass)
+    def self.get_serializer_for(klass, namespace = nil)
       return nil unless config.serializer_lookup_enabled
-      serializers_cache.fetch_or_store(klass) do
+
+      cache_key = ActiveSupport::Cache.expand_cache_key(klass, namespace)
```

Comment:
https://github.com/rails-api/active_model_serializers/blob/v0.10.2/lib/active_model/serializer/caching.rb wow we're behind on releases